<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ELCmdAction</class>
 <widget class="QWidget" name="ELCmdAction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>122</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>50</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>158</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" rowspan="2">
      <widget class="QPushButton" name="Btn_OK">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
	
	
                            color:#ffffff;
                          background-color: rgb(0, 170, 0);
                            border-style: outset;
                            border-width:2px;
                            border-radius:10px;
                            border-color: beige;
                            font: bold 16px;
                            padding:6px;
                            }

  QPushButton:hover{
                            color:#ffffff;
                                background-color: rgb(0, 255, 0);
                            }
                            QPushButton:pressed{
                            color:#ffffff;
                                background-color: rgb(0, 255, 0);
                            }
                            QPushButton:!enabled{
									
								color: rgb(235, 235, 235);
								
	background-color: rgb(143, 143, 143);
							}</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="shortcut">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="2">
      <widget class="QPushButton" name="Btn_NG">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
	
                            color:#ffffff;
                         	 background-color: rgb(255, 0, 0);
                            border-style: outset;
                            border-width:2px;
                            border-radius:10px;
                            border-color: beige;
                            font: bold 16px;
                            padding:6px;
                            }
	

  QPushButton:hover{
                            color:#ffffff;
                                 background-color: rgb(255, 85, 127);
                            }
                            QPushButton:pressed{
                            color:#ffffff;
                                background-color: rgb(255, 85, 127);
                            }
                            QPushButton:!enabled{
									
								color: rgb(235, 235, 235);
								
	background-color: rgb(143, 143, 143);
							}</string>
       </property>
       <property name="text">
        <string>NG</string>
       </property>
       <property name="shortcut">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="BTN_SAVE_EL_DIR">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>EL保存路径</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_7">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton { 
	
                            color:#ffffff;
                         	 background-color: rgb(255, 0, 0);
                            border-style: outset;
                            border-width:2px;
                            border-radius:10px;
                            border-color: beige;
                            font: bold 16px;
                            padding:6px;
                            }
	

  QPushButton:hover{
                            color:#ffffff;
                                 background-color: rgb(255, 85, 127);
                            }
                            QPushButton:pressed{
                            color:#ffffff;
                                background-color: rgb(255, 85, 127);
                            }
                            QPushButton:!enabled{
									
								color: rgb(235, 235, 235);
								
	background-color: rgb(143, 143, 143);
							}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_8">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton { 
	
                            color:#ffffff;
                         	 background-color: rgb(255, 0, 0);
                            border-style: outset;
                            border-width:2px;
                            border-radius:10px;
                            border-color: beige;
                            font: bold 16px;
                            padding:6px;
                            }
	

  QPushButton:hover{
                            color:#ffffff;
                                 background-color: rgb(255, 85, 127);
                            }
                            QPushButton:pressed{
                            color:#ffffff;
                                background-color: rgb(255, 85, 127);
                            }
                            QPushButton:!enabled{
									
								color: rgb(235, 235, 235);
								
	background-color: rgb(143, 143, 143);
							}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_9">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton { 
	
                            color:#ffffff;
                         	 background-color: rgb(255, 0, 0);
                            border-style: outset;
                            border-width:2px;
                            border-radius:10px;
                            border-color: beige;
                            font: bold 16px;
                            padding:6px;
                            }
	

  QPushButton:hover{
                            color:#ffffff;
                                 background-color: rgb(255, 85, 127);
                            }
                            QPushButton:pressed{
                            color:#ffffff;
                                background-color: rgb(255, 85, 127);
                            }
                            QPushButton:!enabled{
									
								color: rgb(235, 235, 235);
								
	background-color: rgb(143, 143, 143);
							}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="shortcut">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton { 
	
                            color:#ffffff;
                         	 background-color: rgb(255, 0, 0);
                            border-style: outset;
                            border-width:2px;
                            border-radius:10px;
                            border-color: beige;
                            font: bold 16px;
                            padding:6px;
                            }
	

  QPushButton:hover{
                            color:#ffffff;
                                 background-color: rgb(255, 85, 127);
                            }
                            QPushButton:pressed{
                            color:#ffffff;
                                background-color: rgb(255, 85, 127);
                            }
                            QPushButton:!enabled{
									
								color: rgb(235, 235, 235);
								
	background-color: rgb(143, 143, 143);
							}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton { 
	
                            color:#ffffff;
                         	 background-color: rgb(255, 0, 0);
                            border-style: outset;
                            border-width:2px;
                            border-radius:10px;
                            border-color: beige;
                            font: bold 16px;
                            padding:6px;
                            }
	

  QPushButton:hover{
                            color:#ffffff;
                                 background-color: rgb(255, 85, 127);
                            }
                            QPushButton:pressed{
                            color:#ffffff;
                                background-color: rgb(255, 85, 127);
                            }
                            QPushButton:!enabled{
									
								color: rgb(235, 235, 235);
								
	background-color: rgb(143, 143, 143);
							}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_6">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton { 
	
                            color:#ffffff;
                         	 background-color: rgb(255, 0, 0);
                            border-style: outset;
                            border-width:2px;
                            border-radius:10px;
                            border-color: beige;
                            font: bold 16px;
                            padding:6px;
                            }
	

  QPushButton:hover{
                            color:#ffffff;
                                 background-color: rgb(255, 85, 127);
                            }
                            QPushButton:pressed{
                            color:#ffffff;
                                background-color: rgb(255, 85, 127);
                            }
                            QPushButton:!enabled{
									
								color: rgb(235, 235, 235);
								
	background-color: rgb(143, 143, 143);
							}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_1">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton { 
	
                            color:#ffffff;
                         	 background-color: rgb(255, 0, 0);
                            border-style: outset;
                            border-width:2px;
                            border-radius:10px;
                            border-color: beige;
                            font: bold 16px;
                            padding:6px;
                            }
	

  QPushButton:hover{
                            color:#ffffff;
                                 background-color: rgb(255, 85, 127);
                            }
                            QPushButton:pressed{
                            color:#ffffff;
                                background-color: rgb(255, 85, 127);
                            }
                            QPushButton:!enabled{
									
								color: rgb(235, 235, 235);
								
	background-color: rgb(143, 143, 143);
							}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="autoRepeatDelay">
            <number>300</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton { 
	
                            color:#ffffff;
                         	 background-color: rgb(255, 0, 0);
                            border-style: outset;
                            border-width:2px;
                            border-radius:10px;
                            border-color: beige;
                            font: bold 16px;
                            padding:6px;
                            }
	

  QPushButton:hover{
                            color:#ffffff;
                                 background-color: rgb(255, 85, 127);
                            }
                            QPushButton:pressed{
                            color:#ffffff;
                                background-color: rgb(255, 85, 127);
                            }
                            QPushButton:!enabled{
									
								color: rgb(235, 235, 235);
								
	background-color: rgb(143, 143, 143);
							}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton { 
	
                            color:#ffffff;
                         	 background-color: rgb(255, 0, 0);
                            border-style: outset;
                            border-width:2px;
                            border-radius:10px;
                            border-color: beige;
                            font: bold 16px;
                            padding:6px;
                            }
	

  QPushButton:hover{
                            color:#ffffff;
                                 background-color: rgb(255, 85, 127);
                            }
                            QPushButton:pressed{
                            color:#ffffff;
                                background-color: rgb(255, 85, 127);
                            }
                            QPushButton:!enabled{
									
								color: rgb(235, 235, 235);
								
	background-color: rgb(143, 143, 143);
							}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="BTN_SAVE_EL_DIR_3">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>误检图像收集</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="BTN_SAVE_EL_DIR_2">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>漏检图像收集</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="lineEdit_ELDIR"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
